testcase/resolve-lock/resolve_lock.go

```diff
@@ -230,8 +231,10 @@ func (c *resolveLockClient) Start(ctx context.Context, cfg interface{}, clientNo
 		// Invoke GC with the safe point
 		greenGCUsed, err := c.resolveLocks(ctx)
 		if err != nil {
-			log.Errorf("[round-%d] failed to run GC at safe point %v", loopNum, c.safePoint)
-			return errors.Trace(err)
+			if !strings.Contains(err.Error(), "region unavailable") {
+				log.Errorf("[round-%d] failed to run GC at safe point %v", loopNum, c.safePoint)
+				return errors.Trace(err)
+			}
```

Comment:
@youjiali1995 Is it OK to check data if `resolveLocks` failed? Should we continue to the next round?

Reply:
> Should we continue to the next round?
Yes.

Reply:
I prefer to retry for some times in a loop. If all retry fails, abort the test.
```go
		var greenGCUsed bool
		for i := 0; i < 5; i++ {
			greenGCUsed, err = c.resolveLocks(ctx)
			if err == nil {
				break
			}
			if !strings.Contains(err.Error(), "region unavailable") {
				log.Errorf("[round-%d] failed to run GC at safe point %v", loopNum, c.safePoint)
				return errors.Trace(err)
			}
		}
		if err != nil {
			log.Errorf("[round-%d] failed to resolve locks for 5 times at safe point %v", loopNum, c.safePoint)
			return errors.Trace(err)
		}
```